pytorch_alternatives/migration_challenge_pytorch_image/Instructions.ipynb

```diff
@@ -4,7 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# REFERENCE SOLUTION: PyTorch MNIST Lift and Shift Exercise\n",
```

Comment:
I'll try and put the "REFERENCE SOLUTION" label back in a merge commit - helps facilitators to easily see whether they have the solution version open before scrolling down!